README.md

```diff
@@ -123,8 +134,11 @@ Copyright &copy; 2019 Kalle Jillheden (jilleJr)
 
 See full copyrights in [LICENSE.md][license.md] inside repository
 
+[json.net-4-unity.converters]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity.Converters
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=5620267)